torch/utils/__init__.py

```diff
@@ -11,3 +11,10 @@ def set_module(obj, mod):
 
 #: Path to folder containing CMake definitions for Torch package
 cmake_prefix_path = _osp.join(_osp.dirname(_osp.dirname(__file__)), 'share', 'cmake')
+
+
+# Re-export submodules
+
+from torch.utils import backcompat as backcompat
+# cannot export data , as this would cause recursive import
+# from torch.utils import data as data
```

Comment:
But you still deleted it from the source module? Might be better to handle the recursive import, but that can be fixed later.